### DISCUSSION TITLE
Worker size limitation effects?

### DISCUSSION THREAD

**Discussion by jozsefm** [flareact/flareact#85] [Q&A] [answered]
I know that the maximum size of a cloudflare worker is 1MB, what happens if my site would create a bigger bundle than that? I'm just evaluating Flareact and it has a unique value proposition but I'm afraid my project will soon get over 1MB and I didn't find any info about that situation in the docs or elsewhere.

**Comment by traffisco** [comment]
Its a real problem with workers. But...
There’s a tendency to include libraries with the intention to use a single function that might be just a simple one-liner. It all adds up.
No solid advice here other than keeping the code clean and minimal. Good practice in general imho.

**Reply by jozsefm** [reply]
I fully aggree with this, but in reality I saw many SPA projects grow to very large bundle sizes. Well over 10MB. So I think it will be something that needs addressing to allow the widest possible adaption for Flareact.

**Comment by traffisco** [comment, marked as answer]
One option is to split different routes to different workers. Data passing with kv storage or localStorage in the browser.

**Reply by jozsefm** [reply]
I was thinking about the same. Do you think it's possible to configure multiple workers to use the same domain but with different paths/routes?

**Reply by traffisco** [reply]
Sure, no problem at all. 
You just set a different route(domain is not important as long its yours and connected to cf) for whatever worker you want or use the same worker for multiple routes/domains.

With some fiddling you can deploy many routes in one go.
